site/source/user-manual/connecting/connecting-tor/tor-os/tor-windows.rst

```diff
@@ -31,11 +31,11 @@ Running Tor on Windows
 
    * In Windows 10, you can simply type ``cmd`` in the Windows search bar, right click on the first result, and select `Run as Administrator`.
 
-#. Once it opens, you can run the following commands, inserting your destination folder (from above) in place of ``<PATH TO>``:
+#. Once it opens, you can run the following commands, inserting your destination folder (from above) between `binPath="` and the `Browser` subfolder, like this:
```

Comment:
If we're doing this, we should guide user where to save to (2 steps above this), instead of saying it doesn't matter (namely, it *does* matter, and needs to be in a system, not user directory).  Maybe we jus list Program Files as the appropriate location, for simplicity?

Reply:
Whoops I missed that.  Will change.

Reply:
I just deleted the step.  Nice catch!